python-backend/app/api/mines/location/models/mine_location.py

```diff
@@ -2,6 +2,8 @@
 import uuid
 
 from sqlalchemy.dialects.postgresql import UUID
+from geoalchemy2 import Geometry, functions, types
+import json
```

Comment:
I see `Geometry` being used, but it looks like `functions`, `types`, and `json` unused imports?